lib/api/index.js

```diff
@@ -5,3 +5,6 @@ module.exports.stream = require('./api-stream')
 module.exports.pipeline = require('./api-pipeline')
 module.exports.upgrade = require('./api-upgrade')
 module.exports.connect = require('./api-connect')
+
+module.exports.responseErrorInterceptor = require('../interceptor/response-error')
+module.exports.interceptors = { retry: require('../interceptor/retry'), error: require('../interceptor/response-error') }
```

Comment:
Sorry, but why are not exporting the interceptor in a similar manner than others?

Reply:
```js
module.exports.interceptors = {
  responseError: require('../interceptor/response-error'),
  retry: require('../interceptor/retry')
}
```

ı am will used this solution, do you think true

Reply:
what you say is correct, it is already exporting itself, there is no need for these codes here  :pray: